pkg/sdk/tags_impl.go

```diff
@@ -149,6 +149,14 @@ func (s *SetTagRequest) toOpts() *setTagOptions {
 			o.column = String(id.Name())
 		}
 	}
+	// TODO(SNOW-1818976): Remove this workaround. Currently ALTER "ORGNAME"."ACCOUNTNAME" SET TAG does not work, but ALTER "ACCOUNTNAME" does.
```

Comment:
Hmm, is this even allowed to modify an account in a different organization? If not, then having only account name seems reasonable.

Reply:
Yes, you can do sth like
```
alter account "ORGNAME"."ACCOUNTNAME" set OAUTH_ADD_PRIVILEGED_ROLES_TO_BLOCKED_LIST=true;
```